bin/install.ps1

```diff
@@ -398,4 +409,15 @@ if ($global:path_advice.Count -gt 0) {
   }
 }
 
+if (-not $shbuild) {
+  Write-Output "Please close this terminal to continue setting up your socket app."
+}
+
+if ($package_setup -eq $true) {
+  $paths = @{}
+  $fso = New-Object -ComObject Scripting.FileSystemObject 
+  $paths["CXX"] = $fso.GetFile($(Get-CommandPath "clang++.exe")).ShortPath
+  ConvertTo-Json $paths > env.json
```

Comment:
maybe we can move this whole block here to a separate file that is used to for this setup in npm

Reply:
@jwerle It needs to be in install.ps1 because `vcvars64.bat` has been executed so `clang++` is in the path. After npm setup completes, the environment is binned and `clang++` is lost until `vcvars64.bat` runs again.